packages/p2p/src/components/orders/popup.jsx

```diff
@@ -101,8 +112,9 @@ class Popup extends Component {
                                 </div>
                             </ThemedScrollbars>
                             <div className='orders__popup-footer'>
+                                {api_error_message && <FormError message={api_error_message} />}
                                 {has_cancel && <Button onClick={onCancel} secondary>{cancel_text}</Button>}
-                                <Button onClick={onClickConfirm} primary>{confirm_text}</Button>
+                                <Button onClick={() => onClickConfirm(this.setApiError)} primary>{confirm_text}</Button>
```

Comment:
why does confirm we set api error, can it be handleConfirm, and check if there is error then we set api error?

Reply:
@oskar-binary as discussed, resolved